src/main/java/com/iota/iri/conf/BaseIotaConfig.java

```diff
@@ -91,6 +92,24 @@
     protected int localSnapshotsDepth = Defaults.LOCAL_SNAPSHOTS_DEPTH;
     protected String localSnapshotsBasePath = Defaults.LOCAL_SNAPSHOTS_BASE_PATH;
 
+    private static BaseIotaConfig config;
+
+    // TODO make this thread safe
+    public static void setInstance(BaseIotaConfig cfg) 
```

Comment:
Static is used to modify the method，and using entity fields inside methods does not cause thread safe.
TODO: private void setInstance(BaseIotaConfig cfg) to package the set and get is it feasible?